project/src/data/technicalArea.js

```diff
@@ -0,0 +1,218 @@
+const getTechnicalAreaData = function() {
+    return [
+        {
+            items: [  
+                {
+                  label: 'Classes',
+                  options: ["0","1","2","3"]
```

Comment:
There's something about this config object that's still needlessly repetitive. Any idea what that is and how one might fix it?

Reply:
options are the same? I thought this data will be sent and we have to deal with it just for now